### DISCUSSION TITLE
I don't see any effect of the changes I make in the vue or style or translation files

### DISCUSSION THREAD

**Discussion by shomid** [chatwoot/chatwoot#8263] [Contributors]
### Describe the bug

I setup chatwoot with docker in development mode.
[I don't see any effect of the changes I make in the vue or style or translation files](https://www.chatwoot.com/docs/contributing-guide/environment-setup/docker)

I dont't see any effect when change in styles or vue or i18n files.

After the changes, I even run the following commands::

`docker-compose build`
`docker-compose run --rm rails bundle exec rails db:chatwoot_prepare`
`docker-compose up`

but I don't see any effect.

### To Reproduce

1. development

### Environment

app.chatwoot.com

### Cloud Provider

None

### Platform

Browser

### Operating system

linux fedora

### Browser and version

chrome 118.0.5993.117

### Docker (if applicable)

. docker version
`Client: Docker Engine - Community
 Version:           24.0.7
 API version:       1.43
 Go version:        go1.20.10
 Git commit:        afdd53b
 Built:             Thu Oct 26 09:09:39 2023
 OS/Arch:           linux/amd64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          24.0.7
  API version:      1.43 (minimum version 1.12)
  Go version:       go1.20.10
  Git commit:       311b9ff
  Built:            Thu Oct 26 09:08:11 2023
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.24
  GitCommit:        61f9fd88f79f081d64d6fa3bb1a0dc71ec870523
 runc:
  Version:          1.1.9
  GitCommit:        v1.1.9-0-gccaecfc
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
`

. docker info
`Client: Docker Engine - Community
 Version:    24.0.7
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.11.2
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.21.0
    Path:     /usr/libexec/docker/cli-plugins/docker-compose
  scan: Docker Scan (Docker Inc.)
    Version:  v0.23.0
    Path:     /usr/libexec/docker/cli-plugins/docker-scan

Server:
 Containers: 78
  Running: 1
  Paused: 0
  Stopped: 77
 Images: 75
 Server Version: 24.0.7
 Storage Driver: btrfs
  Btrfs: 
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 61f9fd88f79f081d64d6fa3bb1a0dc71ec870523
 runc version: v1.1.9-0-gccaecfc
 init version: de40ad0
 Security Options:
  seccomp
   Profile: builtin
  cgroupns
 Kernel Version: 6.5.8-200.fc38.x86_64
 Operating System: Fedora Linux 38 (Workstation Edition)
 OSType: linux
 Architecture: x86_64
 CPUs: 12
 Total Memory: 31GiB
 Name: fedora
 ID: SAUS:ETN3:CAMH:JVAH:JNYE:AJMI:GLZG:2KSA:VHAT:CERF:6JVM:URLL
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Username: shomid
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
`

. docker-compose version
`docker-compose version 1.29.2, build unknown
docker-py version: 5.0.3
CPython version: 3.11.6
OpenSSL version: OpenSSL 3.0.9 30 May 2023
`

**Comment by psilva-sv** [comment]
Did you find a way to solve it? I'm stuck with the same problem

**Reply by Mustafa-Ahmadi55** [reply]
I have same problem, Did you find a solution?

**Comment by prascevic** [comment]
I have same problem.
